frontend2/.env.example

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_CANVAS_IMAGE_URL=https://storage.cloud.google.com/graffio-bucket-3ced5af/images/0xf1b675e890459dfe0a676c01bd14caca20e35634babccc310b49a14c883ea435.png
```

Comment:
Update this to https://storage.googleapis.com/images.testnet.graffio.art/images/0xf1b675e890459dfe0a676c01bd14caca20e35634babccc310b49a14c883ea435.png?